src/server/rpc/procedures/movie-db/movie-db.js

```diff
@@ -201,37 +201,39 @@ var personCredits = function(id, field, subfield) {
 
 MovieDB.serviceName = 'MovieDB';
 
+/**
+ * Find information about a movie
+ * @param {String} title Title of movie
+ */
 MovieDB.searchMovie = function(title) {
     var rsp = this.response;
 
-    if(!title) {
-        rsp.status(400).send('Error: title is not defined');
-    } else {
-        mdb.searchMovie({ query: title }, (err,res) => {
-            if(!err) {
-                rsp.send(res.results.map(e=>e.id));
-            } else {
-                rsp.send(`${err}`);
-            }
-        });
-    }
+    mdb.searchMovie({ query: title }, (err,res) => {
+        if(!err) {
+            rsp.send(res.results.map(e=>e.id));
+        } else {
+            rsp.send(`${err}`);
+        }
+    });
+    
```

Comment:
it is weird this shows up as 4 trailing spaces instead of an empty line but nothing here..

Reply:
I just got rid of the spaces, but that's an interesting feature of GitHub I guess.